x/ibc/04-channel/types/port.go

```diff
@@ -0,0 +1,30 @@
+package types
+
+type Port struct {
+	id string
+}
+
+// ID of the port
+func (p Port) ID() string {
+	return p.id
+}
+
+// // bindPort, expected to be called only at init time
+// // TODO: is it safe to support runtime bindPort?
```

Comment:
Interested in this answer too. Also need to implement this interface I would guess.

Reply:
This will go in the ICS 5 implementation presumably?